github_activity_tracker/services.py

```
@@ -0,0 +1,485 @@
+    language: Language,
+    line_count: int,
+) -> RepoLanguage:
+    """Update line_count for an existing repo–language link."""
+    rl = RepoLanguage.objects.get(repo=repo, language=language)
+    rl.line_count = line_count
+    rl.save()
+    return rl
+
+
+# --- GitCommit, GitHubFile, GitCommitFileChange ---
+from .models import GitCommit, GitHubFile, GitCommitFileChange, FileChangeStatus
```

Ruff (0.15.2)

[error] Redefinition of unused `GitCommit` from line 20: `GitCommit` redefined here

Remove definition: `GitCommit`

(F811)